numpydoc/tests/test_docscrape.py

```diff
@@ -331,10 +331,16 @@ def _strip_blank_lines(s):
 
 
 def line_by_line_compare(a, b):
+    empty_description = NumpyDocString.empty_description  # '..'
```

Comment:
if it's important that these are inserted, why do we want to disregard them in testing?

Reply:
Removed the disregarding of these lines and updated the expected output to account for that.

Reply:
Have you forgotten to commit this?